google-cloud-pubsub/src/main/java/com/google/cloud/pubsub/spi/v1/Subscriber.java

```diff
@@ -119,7 +120,13 @@ private Subscriber(Builder builder) throws IOException {
             Ints.saturatedCast(ackExpirationPadding.getStandardSeconds()));
     clock = builder.clock.isPresent() ? builder.clock.get() : CurrentMillisClock.getDefaultClock();
 
-    flowController = new FlowController(builder.flowControlSettings);
+    flowController =
+        new FlowController(
+            builder
+                .flowControlSettings
+                .toBuilder()
+                .setLimitExceededBehavior(LimitExceededBehavior.ThrowException)
```

Comment:
It is but I think this is the intended behavior. Setting it to `IGNORE` will almost definitely OOM the machine and setting it to `BLOCK` will probably deadlock the executor.

I think we need to either overwrite or check-and-error.